### ISSUE TITLE
Tape recorders don't display the name of whos speaking but do act as universal translators

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#12982]
Tape recorders currently don't display the name of the person saying the message on playback, right?
https://github.com/d3athrow/vgstation13/pull/12731#issuecomment-266233113

They also act as universal translators currently
https://github.com/d3athrow/vgstation13/pull/12731#issuecomment-266252231

Why on earth do we have two copies of the same style file in a cascading file format that is entirely intended to have inheritance?
https://github.com/d3athrow/vgstation13/pull/12731#issuecomment-266456510

In summary:  Tape recorders are badly coded and need your love.  Maintaining the universal translator bit is probably cool.  For an advanced version and not every one.

**Comment by 9600bauds** [comment]
Theoretically speaking what would be cooler

Robert Tables says "butt"

How should the recorder playback this?

>Tape recorder (as Robert Tables) says: "butt"

or

>Robert Tables says: \<darkredtext> "butt" \</darkredtext>

**Comment by unid15** [comment]
How about male voice #1 says: "butt"

I liked that idea

**Comment by Probe1** [comment]
yeah thats fine, just as long as anyone does it.

**Comment by 9600bauds** [comment]
vote 😆 for Tape recorder (as Robert Tables)

vote 😕 for full impersonation, but with a different text style

vote :tada: for Male Voice #2

**Comment by Probe1** [comment]
We're not naming anyone Bobby Tables you little

**Comment by Shadowmech88** [comment]
Male Voice #2 is definitely more noir-y, but you run into the problem where you've got no way to prove who that person is, so the recording can't always be used as evidence.

**Comment by ghost** [comment]
Hold on

**Comment by ghost** [comment]
HOLD ON

**Comment by 9600bauds** [comment]
![nletuno](https://cloud.githubusercontent.com/assets/6526157/21401876/b9a0d5a4-c794-11e6-9ffe-09131deccfe6.jpg)

**Comment by ghost** [comment]
The system for telepathy has it so that if you've heard someone speak in close proximity (or examined them) you know their voice - you could use the "male voice #1" if you've never heard them, or their name at the time if you have

**Comment by ghost** [comment]
Though that'd be a little harder to code so *shrugging woman*

**Comment by 9600bauds** [comment]
And then you don't know who any of the voices in the radio are

and then even at round-end, nobody knows who the xenoarch is when he speaks into the radio

**Comment by Probe1** [comment]
mfw celts crippling addiction

![b18a2738c483b0de6ffdf16a46135451 403x195x1](https://cloud.githubusercontent.com/assets/8550416/21402114/083a9728-c785-11e6-8f24-5901bdc1d4e8.jpg)

**Comment by ghost** [comment]
@9600bauds IMMERSIVE EXPERIENCE

**Comment by kilozombie** [comment]
No no no you replace it with arduous character descriptions instead of names when they haven't revealed their name to you yet with F2
Tall|Hot|Delicious thighs|4 foot breasts|Long flowing cape|Long and troubled past|Her village was burnt down and she was the only survivor|Craves sex|Please go to dorms with me|Blue eyes

**Comment by Probe1** [comment]
I warned you all emojis would lead to liggers and liggers would lead to vore.

**Comment by Kitfox88** [comment]
single robot looking for mommi to polish my chassis iykwim meet me on the solars

**Comment by Intigracy** [comment]
The universal translator function was like the main purpose of their coding in the first place.

**Comment by Probe1** [comment]
I'm calling bullshit on that.  Why wouldn't you call it what i was intended to be then

**Comment by AI-Is-Rogue** [comment]
@Probe1 Aren't they called universal recorders? I thought the name was like that to tip people off that it translates things recorded.

**Comment by west3436** [comment]
Implemented via #31350.
